### PR TITLE
Add ability for refunds on bridging

### PR DESCRIPTION
**Pull request by Drypto13**

### CODE REVIEW

File: contracts/farm/FeeExtractAndDistribute_Arbitrum.sol [hunk 1]
```diff
@@ -36,6 +36,8 @@ contract FeeExtractAndDistribute_Arbitrum is PausableGuardian_0_8 {
 
     address public bridge; //bridging contract
 
+    uint32 public slippage = 10000; //1%
```
Comment (RomanHiden) [inline contracts/farm/FeeExtractAndDistribute_Arbitrum.sol:39]: so you are hardcodding slippage. should it be a mapping per coin with the possibility of change slippage % ?

File: contracts/farm/FeeExtractAndDistribute_Arbitrum.sol [hunk 2]
```diff
@@ -203,4 +205,16 @@ contract FeeExtractAndDistribute_Arbitrum is PausableGuardian_0_8 {
             );
         }
     }
+
+    function setSlippage(uint32 newSlippage) external onlyGuardian {
+        slippage = newSlippage;
```
Comment (RomanHiden) [inline contracts/farm/FeeExtractAndDistribute_Arbitrum.sol:210]: oh I see. but shouldn't slippage be per coin?

File: contracts/farm/FeeExtractAndDistribute_Arbitrum.sol [hunk 3]
```diff
@@ -20,7 +20,7 @@ contract FeeExtractAndDistribute_Arbitrum is PausableGuardian_0_8 {
     address public constant ETH = 0x82aF49447D8a07e3bd95BD0d56f35241523fBab1;
```
Comment (RomanHiden) [inline contracts/farm/FeeExtractAndDistribute_Arbitrum.sol:20]: we really have code duplication. maybe we can extract some into abstract contract ?

File: contracts/farm/FeeExtractAndDistribute_Arbitrum.sol (hunk 2 above)
Comment (Drypto13) [reply contracts/farm/FeeExtractAndDistribute_Arbitrum.sol:210]: we are only bridging USDC

File: contracts/farm/FeeExtractAndDistribute_Arbitrum.sol (hunk 1 above)
Comment (Drypto13) [reply contracts/farm/FeeExtractAndDistribute_Arbitrum.sol:39]: this is slippage for bridging and we are only bridging one token.

File: contracts/farm/FeeExtractAndDistribute_Arbitrum.sol (hunk 3 above)
Comment (Drypto13) [reply contracts/farm/FeeExtractAndDistribute_Arbitrum.sol:20]: we have a couple of lines that are duplicates. Most are just named the same but have different values assigned within the modules.

File: contracts/farm/FeeExtractAndDistribute_Arbitrum.sol (hunk 1 above)
Comment (RomanHiden) [reply contracts/farm/FeeExtractAndDistribute_Arbitrum.sol:39]: shall we have a mapping instead @Drypto13

File: contracts/farm/FeeExtractAndDistribute_Arbitrum.sol (hunk 2 above)
Comment (RomanHiden) [reply contracts/farm/FeeExtractAndDistribute_Arbitrum.sol:210]: oh right

**Review by RomanHiden — APPROVED** [review]
looks good

**Pull request merged**